mobile/multiaccount.go

```diff
@@ -0,0 +1,167 @@
+package statusgo
+
+import (
+	"encoding/json"
+)
+
+// MultiAccountGenerateParams are the params sent to MultiAccountGenerate.
+type MultiAccountGenerateParams struct {
```

Comment:
does it make sense to define all Params only once and use them in mobile and lib modules?

Reply:
looks like the only difference between lib and mobile are types for input and output values. i would consider moving this code to separate module and reusing that code in mobile in lib with simple facade to ensuer proper types (or import mobile from lib if thats possible)